oaknational/Oak-Web-Application
# feat: automatically add the deployment URL to PR descriptions

## Description
Add a post deployment action that will replace the `{owa_deployment_url }` in PR descriptions with the actual URL of the deployment

To get the action working I had to fork it to get it to work outside of the `pull_request` event context (as ours is on `deployment_status`), although I plan to PR the changes back upstream, or if we want we could bring a minimal version of the same into the OWA codebase

I've also changed the template from `{cloud_link}` to `{owa_deployment_url }`

See the PR edit history below (a victim of it's own success, it replaced the one describing what it's supposed to do)
![CleanShot 2023-06-23 at 11 48 18](https://github.com/oaknational/Oak-Web-Application/assets/2717635/c77f5382-ec06-40fd-9ad0-3e087c142d34)


## How to test
1. Branch off of this branch and create a PR including`{owa_deployment_url }` in the body (without the trailing space)
2. See that it's replaced with the deployment URL
